### DISCUSSION TITLE
Unable to generate report

### DISCUSSION THREAD

**Discussion by Marius-CB** [ludeknovy/jtl-reporter#117] [Q&A] [answered]
Hi.

I am unable to generate a report from a Locust csv file.  Below are the logs of "docker-compose logs be"
ttaching to jtl-reporter_be_1
be_1         | 
be_1         | > jtl-reporter-be@1.0.0 start /src/be
be_1         | > node ./dist/server.js
be_1         | 
be_1         | {"level":"info","message":"Express server listening on port 5000"}
be_1         | {"data":null,"isBoom":true,"isServer":false,"level":"error","output":{"headers":{},"payload":{"error":"Unauthorized","message":"The token you provided is invalid","statusCode":401},"statusCode":401}}
be_1         | {"level":"info","message":"GET /api/projects/latest-items 401 89ms","meta":{}}
be_1         | {"data":null,"isBoom":true,"isServer":false,"level":"error","output":{"headers":{},"payload":{"error":"Unauthorized","message":"The token you provided is invalid","statusCode":401},"statusCode":401}}
be_1         | {"level":"info","message":"GET /api/projects 401 132ms","meta":{}}
be_1         | {"data":null,"isBoom":true,"isServer":false,"level":"error","output":{"headers":{},"payload":{"error":"Unauthorized","message":"The token you provided is invalid","statusCode":401},"statusCode":401}}
be_1         | {"level":"info","message":"GET /api/projects/overall-stats 401 104ms","meta":{}}
be_1         | {"level":"info","message":"GET /api/info 200 6ms","meta":{}}
be_1         | {"level":"info","message":"POST /api/auth/initialize-user 201 141ms","meta":{}}
be_1         | {"level":"info","message":"GET /api/info 200 5ms","meta":{}}
be_1         | {"level":"info","message":"POST /api/auth/login 200 124ms","meta":{}}
be_1         | {"level":"info","message":"GET /api/projects 200 14ms","meta":{}}
be_1         | {"code":0,"level":"error","name":"QueryResultError","query":{"text":"SELECT i.id, s.name, environment, project_name as \"projectName\", stat.overview->'startDate' as \"startTime\", status FROM jtl.items as i\n    LEFT JOIN jtl.scenario as s ON s.id = i.scenario_id\n    LEFT JOIN jtl.projects as p ON p.id = s.project_id\n    LEFT JOIN jtl.item_stat as stat ON stat.item_id = i.id\n    WHERE i.report_status = 'ready'\n    ORDER BY start_time DESC LIMIT 10;"},"received":0,"result":{"RowCtor":null,"_parsers":[null,null,null,null,null,null],"_types":{"_types":{"arrayParser":{},"builtins":{"ABSTIME":702,"ACLITEM":1033,"BIT":1560,"BOOL":16,"BPCHAR":1042,"BYTEA":17,"CHAR":18,"CID":29,"CIDR":650,"CIRCLE":718,"DATE":1082,"FLOAT4":700,"FLOAT8":701,"GTSVECTOR":3642,"INET":869,"INT2":21,"INT4":23,"INT8":20,"INTERVAL":1186,"JSON":114,"JSONB":3802,"MACADDR":829,"MACADDR8":774,"MONEY":790,"NUMERIC":1700,"OID":26,"PATH":602,"PG_DEPENDENCIES":3402,"PG_LSN":3220,"PG_NDISTINCT":3361,"PG_NODE_TREE":194,"POLYGON":604,"REFCURSOR":1790,"REGCLASS":2205,"REGCONFIG":3734,"REGDICTIONARY":3769,"REGNAMESPACE":4089,"REGOPER":2203,"REGOPERATOR":2204,"REGPROC":24,"REGPROCEDURE":2202,"REGROLE":4096,"REGTYPE":2206,"RELTIME":703,"SMGR":210,"TEXT":25,"TID":27,"TIME":1083,"TIMESTAMP":1114,"TIMESTAMPTZ":1184,"TIMETZ":1266,"TINTERVAL":704,"TSQUERY":3615,"TSVECTOR":3614,"TXID_SNAPSHOT":2970,"UUID":2950,"VARBIT":1562,"VARCHAR":1043,"XID":28,"XML":142}},"binary":{},"text":{}},"command":"SELECT","duration":15,"fields":[{"columnID":1,"dataTypeID":2950,"dataTypeModifier":-1,"dataTypeSize":16,"format":"text","name":"id","tableID":18197},{"columnID":2,"dataTypeID":1043,"dataTypeModifier":54,"dataTypeSize":-1,"format":"text","name":"name","tableID":18219},{"columnID":3,"dataTypeID":1043,"dataTypeModifier":104,"dataTypeSize":-1,"format":"text","name":"environment","tableID":18197},{"columnID":2,"dataTypeID":1043,"dataTypeModifier":54,"dataTypeSize":-1,"format":"text","name":"projectName","tableID":18215},{"columnID":0,"dataTypeID":3802,"dataTypeModifier":-1,"dataTypeSize":-1,"format":"text","name":"startTime","tableID":0},{"columnID":9,"dataTypeID":18140,"dataTypeModifier":-1,"dataTypeSize":4,"format":"text","name":"status","tableID":18197}],"oid":null,"rowAsArray":false,"rowCount":0,"rows":[]}}
be_1         | {"level":"info","message":"GET /api/projects/latest-items 404 25ms","meta":{}}
be_1         | {"level":"info","message":"GET /api/projects/overall-stats 304 41ms","meta":{}}
be_1         | {"level":"info","message":"POST /api/projects 201 23ms","meta":{}}
be_1         | {"level":"info","message":"GET /api/projects 200 13ms","meta":{}}
be_1         | {"level":"info","message":"GET /api/projects 304 9ms","meta":{}}
be_1         | {"level":"info","message":"GET /api/projects/test 200 9ms","meta":{}}
be_1         | {"level":"info","message":"GET /api/projects/test/scenarios 200 20ms","meta":{}}
be_1         | {"level":"info","message":"POST /api/projects/test/scenarios 201 19ms","meta":{}}
be_1         | {"level":"info","message":"GET /api/projects/test/scenarios 200 13ms","meta":{}}
be_1         | {"level":"info","message":"GET /api/projects 200 8ms","meta":{}}
be_1         | {"level":"info","message":"GET /api/projects/test/scenarios/cats 200 12ms","meta":{}}
be_1         | {"level":"info","message":"GET /api/projects/test/scenarios/cats/notifications 200 15ms","meta":{}}
be_1         | {"level":"info","message":"GET /api/projects/test/scenarios/cats/items?limit=15&offset=0 200 26ms","meta":{}}
be_1         | {"level":"info","message":"GET /api/projects/test/scenarios/cats/trends 200 18ms","meta":{}}
be_1         | {"level":"info","message":"GET /api/projects/test/scenarios/cats 304 53ms","meta":{}}
be_1         | {"level":"info","message":"GET /api/projects/test/scenarios/cats/processing-items 200 57ms","meta":{}}
be_1         | {"level":"info","message":"GET /api/projects/test/scenarios/cats/processing-items 304 9ms","meta":{}}
be_1         | {"level":"info","message":"GET /api/projects/test/scenarios/cats/processing-items 304 8ms","meta":{}}
be_1         | {"level":"info","message":"GET /api/projects/test/scenarios/cats/processing-items 304 8ms","meta":{}}
be_1         | {"level":"info","message":"GET /api/projects/test/scenarios/cats/processing-items 304 9ms","meta":{}}
be_1         | {"level":"info","message":"GET /api/projects/test/scenarios/cats/processing-items 304 10ms","meta":{}}
be_1         | {"level":"info","message":"GET /api/projects/test/scenarios/cats/processing-items 304 10ms","meta":{}}
be_1         | {"level":"info","message":"Starting new item processing for scenario: cats"}
be_1         | {"level":"info","message":"POST /api/projects/test/scenarios/cats/items 200 49ms","meta":{}}
be_1         | {"level":"info","message":"Starting KPI file streaming and saving to db, item_id: 48685739-be02-40e0-9bb3-d65a77583b01"}
be_1         | {"level":"error","message":"Error while parsing data: not a number"}
be_1         | {"level":"error","message":"Error while parsing data: not a number"}
be_1         | {"level":"error","message":"Error while parsing data: not a number"}
be_1         | {"level":"error","message":"Error while parsing data: not a number"}
be_1         | {"level":"error","message":"Error while parsing data: not a number"}
be_1         | {"level":"error","message":"Error while parsing data: not a number"}
be_1         | {"level":"error","message":"Error while parsing data: not a number"}
be_1         | {"level":"error","message":"Error while processing item: 48685739-be02-40e0-9bb3-d65a77583b01: TypeError: Cannot generate an INSERT from an empty array."}
be_1         | {"level":"info","message":"GET /api/projects/test/scenarios/cats/processing-items 200 56ms","meta":{}}
be_1         | {"level":"info","message":"GET /api/projects/test/scenarios/cats/trends 304 57ms","meta":{}}
be_1         | {"level":"info","message":"GET /api/projects/test/scenarios/cats/processing-items 304 10ms","meta":{}}
be_1         | {"level":"info","message":"GET /api/projects/test/scenarios/cats/processing-items 304 9ms","meta":{}}
be_1         | {"level":"info","message":"GET /api/projects/test/scenarios/cats/processing-items 304 8ms","meta":{}}
be_1         | {"level":"info","message":"GET /api/projects/test/scenarios/cats/processing-items 304 10ms","meta":{}}
be_1         | {"level":"info","message":"GET /api/projects/test/scenarios/cats/processing-items 304 8ms","meta":{}}
be_1         | {"level":"info","message":"GET /api/projects/test/scenarios/cats/processing-items 304 8ms","meta":{}}
be_1         | {"level":"info","message":"GET /api/projects/test/scenarios/cats/processing-items 304 10ms","meta":{}}

Attached are the csv files generate by Locust
[cats_exceptions.csv](https://github.com/ludeknovy/jtl-reporter/files/7968832/cats_exceptions.csv)
[cats_failures.csv](https://github.com/ludeknovy/jtl-reporter/files/7968833/cats_failures.csv)
[cats_stats.csv](https://github.com/ludeknovy/jtl-reporter/files/7968834/cats_stats.csv)
[cats_stats_history.csv](https://github.com/ludeknovy/jtl-reporter/files/7968835/cats_stats_history.csv)

Screenshot of Chrome
![image](https://user-images.githubusercontent.com/97291799/151754017-615bc9d2-6f30-4f9c-b024-a399eb0246bb.png)

Please have a look.  I might be doing something wrong as I am totally new to JTL reporter

**Comment by ludeknovy** [comment, marked as answer]
Hello @MariusPlotwis,

JtlReported does not support locust standard CSV output. You need to use a special listener to generate the CSV in a specific format, see: https://jtlreporter.site/docs/integrations/locust
Or you might be uploading the results continuously - https://jtlreporter.site/docs/integrations/locust#continuous-results-uploading


Let me know if you made it work.
